src/redux/products/products.operations.js

```diff
@@ -394,7 +394,7 @@ const addProduct = async (product, upload) => {
 const deleteManyProducts = async (payload) => {
```

Comment:
Maybe, it will be better to rename the function to understand that it removes only one product - `deleteProduct`?